x/gamm/client/cli/tx.go

```diff
@@ -146,66 +143,6 @@ func NewExitPoolCmd() *cobra.Command {
 	return cmd
 }
 
-func NewSwapExactAmountInCmd() *cobra.Command {
```

Comment:
Do you for-see the cli utilizing swap router module directly for the swap? In my mind I thought that we would still support swap calls from gamm that would get sent to the router with the required fields pre-filled. We dont have to address this now, just wanted to bring this up.

Reply:
I think all swap entrypoints should be inside this module. Not sure why we would want to fragment CLI / messages / queries across various modules.